README.md

```diff
@@ -50,7 +50,8 @@ Please send us a pull request with your suggestions.
 
 #### Doesn't this impact the right to free speech? Why are you stomping on my rights?
 
-See this XKCD for our feelings on this matter: https://xkcd.com/1357/
+In the many countries that do have the right to free speech, this means (with obvious exeptions) the government cannot arrest you for saying anything.  Many countries contain free speech in their constitution which can only apply to the government for enforcing or creating laws.
+As most companies and organizations are owned by private individuals, they are only restricted by laws that apply to people and their businesses.  The free speech sections contained in constitutions only apply to the government, restricting the kinds of laws that it can make.  So no, this does not impact your right to free speech.
```

Comment:
I share your concerns @Addvilz.  Github is a private company and they no doubt have a *de jure* right to ignore protecting freedom of expression, but that does not mean they *should.*

Github is quickly becoming the *de facto* place for managing open source code.  The company is increasingly becoming a fixture of our online infrastructure similar to sites like Facebook, Reddit and Google.  Those companies, despite their private nature, have generally strongly allowed for a high standard of free expression to prevail on their platforms, in part because those companies know they have an unwritten social contract with their users to uphold freedom of expression on the platform to levels that match constitutional protections within western societies. 

The major threat of cloud infrastructure is the threat of becoming locked into systems that leave you vulnerable.  With Github openly mocking freedom of expression, that fear is now real.  This may not be apparent to Github executives, but large open source projects have now accumulated years of *crucial* documentation about their codebases within Github's internal code discussion platform.  API startups now routinely rely on Github for customer development and product promotion.  For them, an arbitrary [decision to delete a repository without due process](https://news.ycombinator.com/item?id=9966118) means the systematic destruction of their entire startup company's product & codebase.  

Even worse, if Github disregards free expression, then they their actions become a threat to the open source software community itself.  Many startups and businesses rely on OS software packages created and maintained by other people.  When we begin seeing Github users advocate for the removal of crucial maintainers of open source projects simply because they hold unpopular political views elsewhere, [a trend we are beginning to witness](https://github.com/opal/opal/issues/941) -- I get [worried.](https://en.wikipedia.org/wiki/Bus_factor)